indico/modules/events/contributions/models/contributions.py

```diff
@@ -248,3 +248,11 @@ def can_manage(self, user, role=None, allow_admin=True, check_parent=True, expli
                 session_settings.get(self.event_new, 'coordinators_manage_contributions')):
             return True
         return False
+
+    @classmethod
+    def get_location_for_parent(cls, event):
+        return {'source': event,
```

Comment:
I guess it's a preference. Personally I like it this way, since you can clearly see what the default values are.